astro_2.0/components/ProposalCardRenderer/components/LetterHeadWidget/LetterHeadWidget.tsx

```diff
@@ -30,6 +30,7 @@ const getIconName = (type: ProposalType) => {
       return 'proposalPoll';
     case 'ChangePolicy':
       return 'proposalGovernance';
+    // TODO is that the order we want?
```

Comment:
@karpovdmitry is that something we actually want?

Reply:
i am not sure its actual